<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationForm</class>
 <widget class="QWidget" name="ConfigurationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tiremanagement.qrc">
    <normaloff>:/img/settings.png</normaloff>:/img/settings.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>502</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Database</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>481</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Path</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_Database_Path">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Database_Path">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>481</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Backup</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>Cycle</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Database_BackupCycle">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Database_BackupCycle">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>Path</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Database_BackupPath">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Database_BackupPath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Ok">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="tiremanagement.qrc"/>
 </resources>
 <connections/>
</ui>
